### ISSUE TITLE
TestContainersSuite/TestRunSuccess panic

### ISSUE THREAD

**Issue by bradbeam** [talos-systems/talos#711]
```=== RUN TestContainersSuite
702    === RUN TestContainersSuite/TestRunSuccess
703    panic: runtime error: index out of range [recovered]
704    panic: runtime error: index out of range
705    
706    goroutine 123 [running]:
707    testing.tRunner.func1(0xc0001a5600)
708    /toolchain/go/src/testing/testing.go:830 +0x392
709    panic(0x1b1f820, 0x2aafff0)
710    /toolchain/go/src/runtime/panic.go:522 +0x1b5
711    github.com/talos-systems/talos/internal/pkg/containers_test.(*ContainersSuite).TestRunSuccess(0xc0000e8f60)
712    /src/internal/pkg/containers/containers_test.go:121 +0x12a2
713    reflect.Value.call(0xc00079ca80, 0xc000118b98, 0x13, 0x1cb7602, 0x4, 0xc000030f30, 0x1, 0x1, 0xc000238e68, 0x78, ...)
714    /toolchain/go/src/reflect/value.go:447 +0x461
715    reflect.Value.Call(0xc00079ca80, 0xc000118b98, 0x13, 0xc000238f30, 0x1, 0x1, 0x19c2ad3, 0xe, 0x0)
716    /toolchain/go/src/reflect/value.go:308 +0xa4
717    github.com/stretchr/testify/suite.Run.func2(0xc0001a5600)
718    /toolchain/gopath/pkg/mod/github.com/stretchr/testify@v1.2.2/suite/suite.go:102 +0x2ce
719    testing.tRunner(0xc0001a5600, 0xc000321880)
720    /toolchain/go/src/testing/testing.go:865 +0xc0
721    created by testing.(*T).Run
722    /toolchain/go/src/testing/testing.go:916 +0x35a
723    FAIL    github.com/talos-systems/talos/internal/pkg/containers    2.220s```

**Comment by smira** [comment]
believe a bit of same root cause as #712, but I will also take a look into making it less flaky

**Comment by smira** [comment]
Fixed via #713
